### PR TITLE
Moving custom CLI helper to yargs

### PR DESCRIPTION
**Pull request by fleupold**
This PR replaces the getArgumentsHelper by yargs named arguments in their basic form. Not sure how much effort we need to spend on good documentation and proper usage documentation, which is why I left it pretty light weight for now.

Will sync this with dex-service before merging.

### Test Plan
Getting e2e on dex-services repo run without error.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,16 +33,17 @@
   },
   "homepage": "https://github.com/gnosis/dex-contracts#readme",
   "dependencies": {
+    "@gnosis.pm/mock-contract": "^3.0.7",
```
Comment (bh2smith) [inline package.json:36]: This looks like a dev-dependency to me.

**Review by bh2smith — APPROVED** [review]
This is very nice! Cool stuff Mr. JS

**Comment by fleupold** [comment]
https://github.com/gnosis/dex-services/pull/242 is ready rebased off this branch

**Review by anxolin — COMMENTED** [review]
I'm really happy to see this! 
<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia0.giphy.com%2Fmedia%2F26vUDLOP9YIc8oxGg%2Fgiphy.gif&token=rocket" data-gitmeme-token="rocket"></a>

Have you consider using this for the help?
- https://yargs.js.org/docs/#api-help
- https://yargs.js.org/docs/#api-optionkey-opt
- https://yargs.js.org/docs/#api-demandcount-max-msg-deprecated
https://github.com/yargs/yargs/blob/master/docs/examples.md#yargs-is-here-to-help-you

**Pull request merged**